modules/resource-logs-viewing-cli-console.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * logging/resource-logs-viewing.adoc
+
+[id="resource-logs-viewing-cli-console_{context}"]
+= Viewing resource logs
+
+You can view the log for various resources in the {product-title} CLI and web console. Logs read from the tail, or end, of the log.
+
+.Prerequisites
+
+* Access to the {product-title} Command-line Interface (CLI) also known as `oc`.
+
+.Procedure 
+
+To view resource logs:
+
+* Using the web console
```

Comment:
What do you think about having separate .Procedure sections for the web console and CLI. So instead of the lead in sentence, "To view resource logs:" you can have two different sections:

.Procedure (UI)
and 
.Procedure (CLI)

See [Create the EFS persistent volume claim](https://docs.openshift.com/container-platform/4.7/storage/persistent_storage/persistent-storage-efs.html#efs-pvc_persistent-storage-efs) for an example of this.

Reply:
Ooh, I like that better. I got this formatting from a topic such as [Customizing the project request message](https://docs.openshift.com/container-platform/4.7/applications/projects/configuring-project-creation.html#customizing-project-request-message_configuring-project-creation).

Reply:
@lbarbeevargas We assume folks are smart enough to know at first glance that UI = web console? We don't refer to the web console as _UI_ often,if at all.

Reply:
I think by having "In the _OpenShift Container Platform console_, navigate to ... " in the first step helps to clarify that UI = web console.